source/_integrations/cookidoo.markdown

```diff
@@ -0,0 +1,77 @@
+---
+title: Cookidoo
+description: Instructions on how to integrate the Cookidoo todo list with Home Assistant.
+ha_category:
+  - To-do list
+ha_iot_class: Cloud Polling
+ha_release: 2025.1
+ha_config_flow: true
+ha_codeowners:
+  - '@miaucl'
+ha_domain: cookidoo
+ha_integration_type: service
+ha_platforms:
+  - todo
+related:
+  - docs: /integrations/todo
+    title: To-do list integration documentation
+  - docs: /integrations/#to-do-list
+    title: List of to-do list integrations
+  - docs: /dashboards/todo-list/
+    title: To-do list card
+  - url: https://cookidoo.international/
+    title: Cookidoo the official Thermomix recipe platform
+  - url: https://www.vorwerk.com/
+    title: Vorwerk GmbH
+---
+
+The **Cookidoo** {% term integration %} allows you to interact with your shopping lists of [Cookidoo the official Thermomix recipe platform](https://cookidoo.international/) within Home Assistant.
+
+{% configuration_basic %}
+Email:
+    description: "Enter the email address associated with your Cookidoo account to connect it to Home Assistant."
+Password:
+    description: "Enter the password for your Cookidoo account to enable the connection with Home Assistant."
```

Comment:
Simplified it a little, as connecting to Home Assistant is already kinda implied considering this is the Home Assistant documentation 😉 

```suggestion
Email:
    description: "Enter the email address associated with your Cookidoo."
Password:
    description: "Enter the password for your Cookidoo account."
```